.github/ISSUE_TEMPLATE/bug-template.yaml

```diff
@@ -1,86 +1,28 @@
 name: "\U0001F41B Bug Report"
 description: Report an issue or possible bug
-title: "\U0001F41B BUG:"
+title: "\U0001F41B Bug:"
 labels: ["bug"]
 
 body:
   - type: markdown
     attributes:
       value: |
-        ## Quick Bug Form
-        Thank you for taking the time to file a bug report! Please fill out this form as completely as possible.
-  - type: dropdown
```

Comment:
This is usually wrangler or c3, and when it's not, the user will tell us what the product is. We can make the form less intimidating by removing it.